docs/trex_overview.md

```diff
@@ -29,4 +37,4 @@ The Extensions API and the Embedded API (also known as the [JavaScript API](http
 * You can use the Embedded API for embedding Tableau dashboards in web pages (for example, blog posts), or in line of business applications.
 * You can use the Extensions API for integrating web applications into zones in Tableau dashboards. 
 
-The Extensions API and Embedded API share a similar programming model, but there are differences. The two APIs are not designed to be used together. In the Embedded API, the `Viz` or `VizManager` is the highest-level object. However, in the Extensions API there is no object model or concept for a `Viz` or `VizManager`. For the Extensions API, the workbook is the highest-level object a developer interacts with.
\ No newline at end of file
+The Extensions API and Embedded API share a similar programming model, but there are differences. The two APIs are not designed to be used together. In the Embedded API, the `Viz` or `VizManager` is the highest-level object. However, in the Extensions API there is no object model or concept for a `Viz` or `VizManager`. For the Extensions API, the workbook is the highest-level object a developer interacts with.
```

Comment:
I realize this isn't part of your changes, but few things jumped out to me.. 

i don't think the workbook exists in the extensions api? There are some high level namespaces... ui, dashboard, environement, etc... 

"The two APIs are not designed to be used together." this also seems a bit misleading, theoretically you should be able to embed a viz with a dashboard extension. 

@Kovner thoughts?

Reply:
The working deck PPT for "Explore a whole new world..." TC 18 presentation has a nice list of the differences. 

JavaScript is on the left|Extensions is on the right:

Brings Tableau into other web   applications | Brings other web applications   into Tableau
-- | --
Requires   embedding Tableau dashboard into a web page | Can be used in Desktop, Server, or   embedded dashboard
Custom-built for each embed scenario | Can be made as a re-usable, generic-built dashboard   component
Written in JavaScript | Written in JavaScript (similar calls)

Reply:
@jz-huang @johnDance  Thanks for the feedback on this section. I'll update it with your corrections. I like the table. Thanks for that, John. I'll use it here.  You are right, Jackson. The comparison of the Viz object in JS API and the workbook in the extensions API is off base, and saying "not designed to be used together" is not accurate or very helpful.